testing/adios2/engine/bp/TestBPLargeBlocks.cpp

```diff
@@ -108,6 +130,9 @@ TEST_F(LargeBlocks, MultiBlock)
         }
         reader.Close();
     }
+
+    auto rmCmdStr = std::string("rm -rf ./") + fname;
```

Comment:
I am normally against removing things from the builddir, but given that this test will always generate large files, it appears to be the place to make an exception.

Reply:
Yes, they are platform independent. I was taking liberties here (avoid writing a recursive remove dir function) since we only enable these tests in UNIX-like systems